src/gui/widgets/debugwidget.cpp

```diff
@@ -169,7 +165,20 @@ void DebugWidget::on_importCsvPushButton_clicked()
 
 void DebugWidget::on_debugPushButton_clicked()
 {
-    DEB << AStandardPaths::allPaths()[AStandardPaths::Database	];
+    touchDatabase();
+}
+
+/*!
+ * \brief Acess the database to trigger aDB()::databaseUpdated
+ */
+void DebugWidget::touchDatabase()
+{
+    QMap<QString, QVariant> debugData;
+    debugData.insert("lastname","debugLastName");
+    debugData.insert("firstname","debugFirstName");
+    auto pilot = APilotEntry(1);
+    pilot.setData(debugData);
+    aDB()->commit(pilot);
```

Comment:
Isnt that going to put garbage data in the database?

Reply:
in a way, yes - it resets the logbook owner name to debug values, but since this function is called when the database is reset and filled with testing data for debugging, this is what it's supposed to do as well.

Reply:
i believe it goes also to [this](https://github.com/fiffty-50/openpilotlog/pull/43#discussion_r551342980).
The "refresh database" functionality can be decoupled from all database operations. So commit calls `refresh` or whatever its called. That way debug can just call refresh which does just that.

Reply:
fixed